utils.go

```diff
@@ -135,3 +136,8 @@ func copyFile(src, destDir string) (string, error) {
 
 	return dest, nil
 }
+
+func isLocalPath(location string) bool {
```

Comment:
Change this from just `isLocalPath` to `isAbsLocalPath`, as the usage of this goes along with another check of `filepath.IsAbs`, thereby simplifying the use as the if condition is only followed when both conditions are true.